### ISSUE TITLE
[Scheduler] Cannot read property 'left' of undefined error while dragging an event

### ISSUE THREAD

**Issue by stnikolova** [telerik/kendo-angular#3307]
**Describe the bug**
The following error is raised while dragging an event:

`TypeError: Cannot read property 'left' of undefined`

**To Reproduce**
1. Create an all-day event.
2. Grab that event and begin dragging.
3. A semi-transparent preview (drag cue) of the dragged event appears only after you drag it a certain distance.
4. The trick is to drag the event above the all-day container **before** the drag cue has appeared. That is achieved more easily if you grab the event near its top border.

**Screen recording of the reproduction:**

https://user-images.githubusercontent.com/45083324/119463675-a6498e00-bd4a-11eb-990f-399e124a5374.mp4

**Comment by rsseurope** [comment]
Maybe interesting to know is that we have this issue happening a lot and we are hiding the all day area with css styling:
```
.k-header.k-scheduler-footer {
  display: none;
}

.k-scheduler-toolbar.k-header.k-floatwrap {
  display: none;
}

.k-scheduler-table tr:nth-of-type(0), .k-scheduler-times-all-day, .k-scheduler-table.k-scheduler-header-all-day {
  display: none;
}

.k-calendar .k-calendar-header .k-today {
  display: none;
}
```
link to Sentry exception log of the bug: https://sentry.io/share/issue/0fe3bb3d99ee467cada45b5e09c766f1/

**Comment by rsseurope** [comment]
Any progress on this issue?

**Comment by gkarapeev** [comment]
Hi Tonie,

A fix is available in the latest development build of the Scheduler package and it is currently under testing.
If it successfully passes testing, it will be rolled out with the next version.

Regards,
Georgi

**Comment by ipeshev** [comment]
A fix shipped with with v2.1.3.
